### PR TITLE
[IMP] product: allow reordering product tags in the backend

### PR DESCRIPTION
**Pull request by ltinel**
The purpose of this change is to allow customizing the order of the product tags on the website's product page (which uses the same order as the backend).

This change also removes the "Configure tags" action from the product and variant forms. Indeed, this action opens a "Product Tags" view that isn't specific to the product or variant, which is confusing. Product tags can be accessed from Website > eCommerce > Product Tags.

task-3440866

### CODE REVIEW

File: addons/product/models/product_tag.py [hunk 1]
```diff
@@ -15,6 +16,7 @@ def _get_default_variant_id(self):
         return self.env['product.product'].browse(self.env.context.get('product_variant_id'))
 
     name = fields.Char(string="Name", required=True, translate=True)
+    sequence = fields.Integer(string='Sequence')
```
Comment (vava-odoo) [inline addons/product/models/product_tag.py:19]: Shouldn't there be a default? Or it is ok to be the first one after creation?
Also, I know examples around it don't follow this remark but there is no need to define the string if it is the same as the name of the field.
```suggestion
    sequence = fields.Integer()
```

**Review by vava-odoo — CHANGES_REQUESTED** [review]
Hi @ltinel 
Looks good. Just a question and two small remarks.
Cheers

File: addons/product/models/product_product.py [hunk 2]
```diff
@@ -319,7 +319,7 @@ def _compute_product_document_count(self):
     @api.depends('product_tag_ids', 'additional_product_tag_ids')
     def _compute_all_product_tag_ids(self):
         for product in self:
-            product.all_product_tag_ids = product.product_tag_ids | product.additional_product_tag_ids
+            product.all_product_tag_ids = (product.product_tag_ids | product.additional_product_tag_ids).sorted(lambda t: t.sequence)
```
[the remark this answers is not in the document]
Comment (ltinel) [reply addons/product/models/product_product.py]: Done :)

**Review by vava-odoo — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**